frontend/src/components/pages/CardDetails/Presentation.jsx

```diff
@@ -19,33 +19,66 @@ import AssigneesList from "../../widgets/AssigneesList";
 import ReviewersTable from "../../widgets/ReviewersTable";
 import Reviews from "./Reviews";
 
-const useStyles = makeStyles((theme) => {
-  return {
-    paper: {
-      padding: theme.spacing(1, 2, 1),
-    },
+const TableCell = withStyles({
+  root: {
+    borderBottom: "none",
+    padding: "3.24%",
+  },
+})(MuiTableCell);
 
-    commentColumn: {
-      minWidth: 300,
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
     },
-
-    tableContainer: {
-      maxHeight: 200,
+    paper: {
+      padding: theme.spacing(2),
+      height: "100%",
+      textAlign: "auto",
+      color: theme.palette.text.primary,
+      boxSizing: "border-box",
+      maxHeight: 350,
     },
-    sectionPaper: {
+    gridItem: {
+      boxSizing: "border-box",
```

Comment:
Wont the grid spacing prop work here?

https://v4.mui.com/components/grid/#spacing